### DISCUSSION TITLE
Need help for renewing github student developer pack

### DISCUSSION THREAD

**Discussion by WaterColor201** [community/community#112567] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/110291198/bfff4571-38ab-44b7-93bd-bc0e4cc92dac)
I'm a college student,I have been acessed at my first time.At this time I applied,I have been modified following the quest,but the problems arise in above picture.I really need the student pack,thank you for helping me.

**Comment by adarsh-site** [comment]
try submitting a document which has your name as well as college name clearly mentioned.

**Comment by samus-aran** [comment]
Hi @WaterColor201 👋 

We can see that your academic benefits have been granted, congratulations! Enjoy! 🎉
